contracts/src/NStates.sol

```diff
@@ -49,6 +49,7 @@ contract NStates is IncrementalMerkleTree {
     IVerifier verifierContract =
         IVerifier(0xe7f1725E7734CE288F8367e1Bb143E90bb3F0512);
 
+    event NewMove(uint256 hUFrom, uint256 hUTo);
     event NewLeaf(uint256 h);
```

Comment:
why do we still need NewLeaf()

Reply:
at this point we do not. I figured there might be some future use case, but it's easy enough to put it back in if needed

Reply:
sg !